samples/interactive-tutorials/product/setup_product/products_create_gcs_bucket.py

```diff
@@ -23,7 +24,9 @@
 bucket_name = f"{project_id}_products_{timestamp_}"
 
 create_bucket(bucket_name)
-upload_blob(bucket_name, "../resources/products.json")
-upload_blob(bucket_name, "../resources/products_some_invalid.json")
+
+path_to_resources_dir = f"{Path.home()}/cloudshell_open/python-retail/samples/interactive-tutorials/resources/"
```

Comment:
Hard-coding the absolute path like this is making the assumption that the users of setup_product will only ever run this file in cloudshell. I actually like the relative path system before this change, because it does not make this assumption.

Reply:
this is our problem. we don't know exactly from which directory our sample would be run from.